src/webgpu/api/validation/createBindGroup.spec.ts

```diff
@@ -784,7 +787,12 @@ g.test('buffer,usage')
       usage,
     });
 
-    const isValid = GPUBufferUsage.UNIFORM & usage;
+    let isValid = false;
+    if (type === 'uniform') {
+      isValid = GPUBufferUsage.UNIFORM & usage ? true : false;
+    } else if (type === 'storage' || type === 'read-only-storage') {
+      isValid = GPUBufferUsage.STORAGE & usage ? true : false;
```

Comment:
Perhaps use an explicit cast: `Boolean(usage & GPUBufferUsage.STORAGE)`

Reply:
Let me merge this PR now. Please let me know if it should be fixed. Let me update it in the following PR together.

Reply:
No worries, let's just leave as is. LGTM!